kura/org.eclipse.kura.driver.block/src/main/java/org/eclipse/kura/driver/block/task/AbstractBlockDriver.java

```diff
@@ -0,0 +1,320 @@
+/*******************************************************************************
```

Comment:
I've added a few more tests that should cover the remaining parts of the bundle, can you please have a look to see if they are fine?
Feel free to modify/extend the tests if you want.